compute/module/sub-modules/qprovisioner/scripts/provision.py

```diff
@@ -252,7 +252,7 @@ def chkurl(url: str, no_sec: bool = False) -> bool:
         k = "k" if no_sec else ""
         cmd = f'curl -sL{k} -w "%%{{http_code}}\\n" "{url}" -o /dev/null --connect-timeout 10 --retry 3 --retry-delay 5 --max-time 60'
         result = run_command(cmd, timeout=70, check=False)
-        return result.stdout.strip() == "200"
+        return result.stdout.strip()=="200"
```

Comment:
Can we undo this formatting change? Our coding standard is to leave spaces around comparators